app/src/main/java/com/stevesoltys/seedvault/transport/backup/PackageService.kt

```diff
@@ -154,6 +154,11 @@ internal fun PackageInfo.isNotUpdatedSystemApp(): Boolean {
 
 internal fun PackageInfo.doesNotGetBackedUp(): Boolean {
     if (packageName == MAGIC_PACKAGE_MANAGER || applicationInfo == null) return true
-    return applicationInfo.flags and FLAG_ALLOW_BACKUP == 0 && // does not allow backup
+    return applicationInfo.flags and FLAG_ALLOW_BACKUP == 0 || // does not allow backup
```

Comment:
This two character change is the core of the fix. The rest is just resolving collateral damage from not properly differentiating between `FLAG_ALLOW_BACKUP` and `FLAG_STOPPED`.

Reply:
Worth splitting out in a separate commit perhaps?

Reply:
I split out this commit and clarified the other changes in the commit message of the second commit.